e2e/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The OWASP Coraza contributors
+# SPDX-License-Identifier: Apache-2.0
+
+FROM curlimages/curl
+USER root
+
+WORKDIR /workspace
+
+RUN apk add --no-cache bash
+
+COPY ./e2e-example.sh /workspace/e2e-example.sh
+
+ENV ENVOY_HOST=envoy:8080
+ENV HTTPBIN_HOST=httpbin
+
+ENTRYPOINT ["bash"]
+CMD ["-c", "/workspace/e2e-example.sh envoy"]
```

Comment:
`envoy` is arbitrary, probably you want to rewrite the command in docker-compose where you know what name will work for the envoy instance.